wordle-web/pages/about.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div>
+    <div class="text-box">
+      <p>{{ sentence }}</p>
+    </div>
+    <v-btn icon="mdi-arrow-left" @click="router.push('/')"></v-btn>
```

Comment:
```suggestion
    <v-btn icon="mdi-arrow-left" @click="router.push('/')"/>
```
Use self closing tag

Reply:
Fixed